### PR TITLE
Run tests with multiple solc versions

### LINKED ISSUE #96: Extend testing infrastructure
Should run tests using different compiler versions.

### PR DESCRIPTION
**Pull request by axic**
Findings:
- 0.4.26 works
- 0.3.6 works
- 0.2.1 works
- 0.2.0 works
- 0.1.6 works
- 0.1.1 works

Closes #96.

### CODE REVIEW

File: wrapper.js [hunk 1]
```diff
@@ -114,8 +114,8 @@ function setupMethods (soljson) {
       readCallback = readCallback.import;
     }
 
-    if (readCallback !== undefined && typeof readCallback !== 'function') {
-      return formatFatalError('Invalid import callback supplied');
+    if (readCallback !== undefined) {
+      assert(typeof callback === 'function', 'Invalid callback specified.');
```
Comment (axic) [inline wrapper.js]: Typo.

File: test/compiler.js [hunk 2]
```diff
@@ -568,3 +578,23 @@ tape('API backwards compatibility', function (t) {
...
+  'v0.1.6+commit.d41f8b7',
+  'v0.2.0+commit.4dc2445',
+  'v0.2.1+commit.91a6b35',
+  'v0.3.6+commit.3fc68da',
+  'v0.4.26+commit.4563c3fc'
+];
+for (var version in versions) {
+  version = versions[version];
+  console.log(version);
+  execSync(`curl -o /tmp/${version}.js https://ethereum.github.io/solc-bin/bin/soljson-${version}.js`);
```
[the remark this answers is not in the document]
Comment (leonardoalt) [reply test/compiler.js]: Looks good to me. Test still failing though

[the remark this answers is not in the document]
Comment (axic) [reply test/compiler.js]: The helpers were outside of the `runTests` scope and they've referenced the wrong `solc`...

**Comment by leonardoalt** [comment]
node4 is still complaining about scope I think

**Pull request merged**